File: Colleen/Colleen.c
#include <stdio.h>

void function1()
{
    char *s = "#include <stdio.h>%c%cvoid function1()%c{%c    char *s = %c%s%c;%c    printf(s,10,10,10,10,34,s,34,10,10,10,10,10,10,10,10,10,10,10,10,10,10);%c}%c%c/*%c    comments outside main%c*/%c%cint main()%c{%c    /*%c        comments inside main%c    */%c    function1();%c}";
    printf(s,10,10,10,10,34,s,34,10,10,10,10,10,10,10,10,10,10,10,10,10,10);
}

/*
    comments outside main
*/

int main()
{
    /*
        comments inside main
    */
    function1();
}

File: PY/Colleen/Colleen.py
def function1():
    s = ("def function1():{2}    s = ({1}{0}{1}){2}    print(s.format(s, '{3}{1}', '{3}n', '{3}{3}')){2}{2}# comments outside main{2}{2}if __name__ == {1}__main__{1}:{2}    # comments inside main{2}    function1()")
    print(s.format(s, '\"', '\n', '\\'))

# comments outside main

if __name__ == "__main__":
    # comments inside main
    function1()
